src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/mobile/perform_interaction_test.cc

```diff
@@ -254,6 +257,49 @@ TEST_F(PerformInteractionRequestTest,
                             "UI is not supported by system");
 }
 
+TEST_F(PerformInteractionRequestTest, ChoiceProcessesOnHHMI_REJECT) {
+  MessageSharedPtr message = CreateMessage(smart_objects::SmartType_Map);
+  (*message)[strings::params][strings::connection_key] = kConnectionKey;
+  (*message)[strings::msg_params][strings::interaction_choice_set_id_list][0] =
+      kChoiceSetId;
+  smart_objects::SmartObject* choice_set_id =
+      &((*message)[am::strings::msg_params]
+                  [am::strings::interaction_choice_set_id]);
+  std::shared_ptr<PerformInteractionRequest> command =
+      CreateCommand<PerformInteractionRequest>(message);
+
+  smart_objects::SmartObject& response = *message;
+
+  response[strings::params][strings::message_type] = am::MessageType::kResponse;
+  response[strings::params][strings::correlation_id] = kCorrelationId;
+  response[strings::params][strings::protocol_type] = 0;
```

Comment:
@IGapchuk why do you need this?

Reply:
@AKalinich-Luxoft Deleted in fbc582d9bf3feaa96a345733d977106eea7a02b4.